examples/pod_failure.yaml

```diff
@@ -0,0 +1,18 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+apiVersion: chaos.datadoghq.com/v1beta1
+kind: Disruption
+metadata:
+  name: pod-failure
+  namespace: chaos-demo
+spec:
+  selector:
+    app: demo-curl
+  containers: # only target the dummy container, you can specify multiple containers here (all containers are targeted by default)
+    - dummy
+  count: 1
+  podFailure:
+    kill: false # send a SIGINT signal to the pod's containers
```

Comment:
Are we sure `SIGINT` is preferable to `SIGTERM`? If I understand correctly (and that's a big if), `SIGINT` is primarily for `ctrl+c` style interrupt, while `SIGTERM` is the more general "tell a process to finish", yes?

Reply:
Correct. Following my last comment, I am changing this to `SIGTERM` and `SIGKILL`. I don't think SIGINT brings too much value here.